### PR TITLE
Remove Dropdown props from DownloadButton that are never used

### PR DESCRIPTION
**Pull request by josejulio**
These are overriden by DownloadButton and are never used - no point in accepting them.

### CODE REVIEW

**Review by Hyperkid123 — COMMENTED** [review]
Rather than removing it, can we just pass it to the component? In case anyone wants to change the variant?

**Comment by josejulio** [comment]
`DownloadButton` component manually specifies a `toggle`, `dropdownItems` and `isPlain`. 

Which ones would you like to pass trough if they are specified? All or only `isPlain` ?

**Comment by Hyperkid123** [comment]
@josejulio only the `isPlain`. the rest depends on internal variables.

**Comment by josejulio** [comment]
@Hyperkid123  done

File: packages/components/src/DownloadButton/DownloadButton.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const DownloadButton: React.FunctionComponent<DownloadButtonProps> = ({
       {conditionallyTooltip(
         <Dropdown
           {...props}
-          isPlain
+          isPlain={props.isPlain ?? true}
```
Comment (rvsia) [inline packages/components/src/DownloadButton/DownloadButton.tsx:51]: We can move `isPlain` before `{...props}` so consumers can freely overwrite it.

Comment (rvsia) [reply packages/components/src/DownloadButton/DownloadButton.tsx:51]:
```jsx
<Dropdown 
  isPlain
  {...props}
  ...
  />
```

**Pull request merged**